src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts

```diff
@@ -1,4 +1,4 @@
-import { AfterViewInit, Component, ElementRef, Input, OnDestroy, ViewChild } from '@angular/core';
+import { AfterViewInit, Component, ElementRef, HostBinding, Input, OnDestroy, ViewChild } from '@angular/core';
```

Comment:
I believe you don't need to import this `HostBinding`

Reply:
Done